### PR TITLE
Enhancement: Add help links in Houdini menu.

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
Add help links in Houdini menu 
I've added 5 links: 
- Ayon server
- [Houdini Artist Docs](https://ayon.ynput.io/docs/addon_houdini_artist)
- [Houdini Admin Docs](https://ayon.ynput.io/docs/addon_houdini_admin)
- [Community Forums](https://community.ynput.io/)

image

## Testing notes:
1. Launch Houdini
2. Test links

### CODE REVIEW

**Review by moonyuet — APPROVED** [review]
Looks Good. Maybe we can discuss possibly how to improve how `About Ynput` shows, (similar to About Houdini).
image

**Comment by MustafaJafar** [comment]
> Looks Good. Maybe we can discuss possibly how to improve how `About Ynput` shows, (similar to About Houdini). image

Let's summon @mkolar @antirotor !

**Comment by antirotor** [comment]
Looks good! With the `About Ynput` - I am not sure we need some company information directly inside DCC? If it would be `About AYON`, would it show something different than clicking on the version in tray?

image

**Comment by MustafaJafar** [comment]
> If it would be `About AYON`, would it show something different than clicking on the version in tray?

I forgot we have that! 
I think there is nothing to do about it.. I believe the button in the tray is enough.

--- 
Instead, Maybe we can have `About Houdini Addon` that shows the current Houdini addon version and a link to the features page in the docs.

File: openpype/hosts/houdini/startup/MainMenuCommon.xml [hunk 1]
```diff
@@ -104,6 +104,39 @@ parent = hou.qt.mainWindow()
 host_tools.show_experimental_tools_dialog(parent)
 ]]></scriptCode>
             </scriptItem>
+
+        <separatorItem/>
+
+        <scriptItem id="ayon_server">
+            <label>Open Ayon server in browser</label>
```
Comment (iLLiCiTiT) [inline openpype/hosts/houdini/startup/MainMenuCommon.xml:111]:
```suggestion
            <label>Open AYON server in browser</label>
```

**Comment by BigRoy** [comment]
> image


Out of curiosity, why does the AYON label show far-left as opposed to far-right in the menu for you?

**Comment by MustafaJafar** [comment]
> Out of curiosity, why does the AYON label show far-left as opposed to far-right in the menu for you?

I wish I could answer 🫣

**Comment by mkolar** [comment]
Because we're splitting OpenPype into ayon-core and individual host addons, this PR would have to be re-created to target one of those.

We're closing it down, but we'll he happy for a new PR to [ynput/ayon-core](https://github.com/ynput/ayon-core) or the host addon repository once it's up.

**Pull request closed without merging**